python/mead/tasks.py

```diff
@@ -355,15 +355,21 @@ def _create_backend(self):
             backend.params = {'pc': _dynet.ParameterCollection(), 'batched': batched}
         elif backend.name == 'tf':
             # FIXME this should be registered as well!
-            from mead.tf.exporters import ClassifyTensorFlowExporter
-            backend.exporter = ClassifyTensorFlowExporter
+            exporter_type = kwargs.get('exporter_type', 'default')
```

Comment:
If we had folders for `mead/pytorch` and `mead/dy` that are empty (because we don't support export yet) could we replace this with a `load_user_module('mead.{backend}')` in the `backend.load` call? This should load the registries that the `create_exporter` call in `mead-export` uses

Reply:
i dont think that's a necessary change. WDYT @dpressel

Reply:
I don't think its necessary and can be merged without it but it seems like what will happen moving forward especially as pytorch gets more production features.